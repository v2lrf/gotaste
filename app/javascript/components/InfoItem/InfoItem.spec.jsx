import React from 'react'
import { InfoItem } from '.'

describe('InfoItem', () => {
  let component
  let props
  beforeEach(() => {
    props = {
      imageSrc: 'https://via.placeholder.com/150',
      headline: 'I am a headline',
      tagline: 'I am a tagline'
    }

    component = shallow(<InfoItem {...props} />)
  })

  describe('Snapshots', () => {
    it('should match the regular snapshot', () => {
      expect(component).toMatchSnapshot()
    })

    it('should match the large snapshot', () => {
      component.setProps({
        ...props,
        large: true
      })
      expect(component).toMatchSnapshot()
    })
  })
})
