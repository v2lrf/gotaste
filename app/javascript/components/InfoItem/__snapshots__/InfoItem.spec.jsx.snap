// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoItem Snapshots should match the large snapshot 1`] = `
<div
  className="flex items-center"
>
  <figure
    className="flex self-start p-2 mr-2 bg-white shadow rounded-full w-16 h-16"
  >
    <img
      alt=""
      className="rounded-full"
      src="https://via.placeholder.com/150"
    />
  </figure>
  <div
    className="flex-grow"
  >
    <div
      className="font-bold text-xl"
    >
      I am a headline
    </div>
    <div
      className="text-grey-darker text-base"
    >
      I am a tagline
    </div>
  </div>
</div>
`;

exports[`InfoItem Snapshots should match the regular snapshot 1`] = `
<div
  className="flex items-center"
>
  <figure
    className="flex self-start p-2 mr-2 bg-white shadow rounded-full w-12 h-12"
  >
    <img
      alt=""
      className="rounded-full"
      src="https://via.placeholder.com/150"
    />
  </figure>
  <div
    className="flex-grow"
  >
    <div
      className="font-bold text-base"
    >
      I am a headline
    </div>
    <div
      className="text-grey-darker text-sm"
    >
      I am a tagline
    </div>
  </div>
</div>
`;
